### PR TITLE
Use shrinkwrap to create a repeatable build

### PR DESCRIPTION
**Pull request by someword**
**What I did**:
add shrinkwrap
**How I did it**:
painfully
**Why**:
so the builds will work

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -50,10 +50,6 @@ workflows:
             - sb/normalize
       - sb/publish-artifacts:
           context: project-bot
-          filters:
```
Comment (figadore) [inline .circleci/config.yml:53]: why is this removed? we still want to deploy branches to tools-dev, right?

Comment (someword) [reply .circleci/config.yml:53]: without the filter and https://github.com/cambiahealth/tools-env-configs/pull/67/files i thought we would be able to deploy branches to tools-dev? Doesn't the removal of the filter all branchs builds to be published to artifactory?

Comment (someword) [reply .circleci/config.yml:53]: @figadore - reese am i missing something with how filters work? I'd like to get this merged today to get this service into the next round of scans over the weekend.

**Review by figadore — APPROVED** [review]
